### PR TITLE
netdev: unify msg type macro with msg type macros in netapi

### PR DESCRIPTION
**Pull request by authmillenon**

### CODE REVIEW

**Comment by authmillenon** [comment]
Rebased to #2428

File: sys/include/net/ng_netdev.h [hunk 1]
```diff
@@ -121,8 +121,8 @@ typedef struct {
      * @return              -EOVERFLOW if available space in *value* given in
      *                      *value_len* is too small to store the option value
      */
-    int (*get)(ng_netdev_t *dev, ng_netconf_opt_t opt,
-                      void *value, size_t *value_len);
```
Comment (haukepetersen) [inline sys/include/net/ng_netdev.h]: acutally I purposely put value and value_len in the same row, as they kind of belong together...

**Comment by haukepetersen** [comment]
#2428 is merged. You could consider my comment on the line break changes, but anyhow: ACK when rebased.

**Comment by haukepetersen** [comment]
also adjusted #2426 to cope with this PR

**Comment by authmillenon** [comment]
Considered comment and waiting for Travis… don't understand why I need to rebase.

**Comment by LudwigOrtmann** [comment]
> don't understand why I need to rebase

Me neither.

**Comment by authmillenon** [comment]
Kicked Travis… for the third time…

**Pull request merged**